x-pack/plugins/data_enhanced/server/search/eql_search_strategy.ts

```diff
@@ -64,7 +67,7 @@ export const eqlSearchStrategyProvider = (
         (response) => response.body.id,
         request.id,
         options
-      ).pipe(utils.toKibanaSearchResponse());
+      ).pipe(normalizeEqlResponse(), utils.toKibanaSearchResponse());
```

Comment:
This is the crux of the fix; details are in 9e5abf432f978f98ecdeba53f48bc7f73a9610e7